cli/utilities/packages.py

```diff
@@ -185,7 +187,11 @@ def register(self, username, password, serverurl=None, baseurl=None, force=False
             serverurl (str): server URL
             baseurl (str): base URL for content
             force (bool): register the system even if it is already registered
+            clean (bool): if True, clean local subscription data before registering.
         """
+        if clean:
```

Comment:
I would suggest to keep clean method separate, lets not intercall unless required it or cannot be handled from caller.
And mainly handle this clean --> register --> enable repos --> so on... on the calling module or workflow, not on these framework lib or utilities.

Reply:
done